test/util.go

```diff
@@ -0,0 +1,85 @@
+package test
```

Comment:
Seems like the helpers in this file are only used within `test/database/user_test.go`?

You could just move it to `test/database/utils_test.go` in that case. That way it's also not part of the production binary.

Reply:
I didn't want to do that because `utils_test.go` looks like something that's testing a `utils.go` file instead of providing helpers. But I don't want those methods in the bin for no reason.